opm/simulators/linalg/bda/BILU0.cpp

```diff
@@ -244,6 +356,102 @@ bool BILU0<block_size>::create_preconditioner(BlockedMatrix *mat)
     return true;
 } // end create_preconditioner()
 
+template <unsigned int block_size>
+bool BILU0<block_size>::create_preconditioner(BlockedMatrix *mat, BlockedMatrix *jacMat)
```

Comment:
Can't we get along with one create_preconditioner method and some if statements? This seems to be a lot of duplicate code.